### PR TITLE
experimental: Paste from webflow 4

### PR DESCRIPTION
**Pull request by kof**
## Description

- webflow css preset

## Steps for reproduction

1. click button
2. expect xyz

## Code Review

- [ ] hi @kof, I need you to do
  - conceptual review (architecture, feature-correctness)
  - detailed review (read every line)
  - test it on preview

## Before requesting a review

- [ ] made a self-review
- [ ] added inline comments where things may be not obvious (the "why", not "what")

## Before merging

- [ ] tested locally and on preview environment (preview dev login: 5de6)
- [ ] updated [test cases](https://github.com/webstudio-is/webstudio/blob/main/apps/builder/docs/test-cases.md) document
- [ ] added tests
- [ ] if any new env variables are added, added them to `.env` file

### CODE REVIEW

File: apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts [hunk 1]
```diff
@@ -0,0 +1,7212 @@
...
+    },
+    {
+      property: "display",
+      value: {
+        type: "keyword",
+        value: "block",
+      },
+    },
+  ],
+  "w-file-upload-uploading-btn": [
```
Comment (TrySound) [inline apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts:2657]: what are these? Do not look like tags

Comment (kof) [reply apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts:2657]: the keys are either tags or classes, I will start using them in the next prs

Comment (kof) [reply apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts:2657]: its how webflow defines styles on their custom abstractions like a link button etc

Comment (TrySound) [reply apps/builder/app/shared/copy-paste/plugin-webflow/__generated__/style-presets.ts:2657]: they are not copied to clipboard too?

**Comment by TrySound** [comment]
I think it makes sense to not parse values prematurely only on demand. This will reduce the size of generated module a lot and reduce loading time. Values is a more straightforward piece to parse.

**Comment by kof** [comment]
> I think it makes sense to not parse values prematurely only on demand. This will reduce the size of generated module a lot and reduce loading time. Values is a more straightforward piece to parse.

We will see in the later PRs how much is actually unsused in the end, seems premature right now

**Comment by kof** [comment]
Also if doing it lazy, then we need to load the CSS file instead of prebuilding and that's what I mentioned earlier the difference between 40 vs 120kb

**Comment by kof** [comment]
Additional downside of runtime parsing is if there is something wrong being generated its not immediately obvious

**Comment by TrySound** [comment]
I mean specifically leave values unparsed. Not whole structure. So it is still easy to access them.

**Review by TrySound — APPROVED** [review]

**Pull request merged**